### PR TITLE
Added I don't knows to source, referrer, and trust questions; updated te...

### PR DESCRIPTION
**Pull request by moduloprime**

### CODE REVIEW

**Comment by moduloprime** [comment]
@adrifelt PTAL

File: extension/surveys/survey-generator.js [hunk 1]
```diff
@@ -448,8 +448,11 @@ function getDomNameFromValue(answer) {
  * @returns {Array.Object} The randomized array.
  */
 function knuthShuffle(arr, randomType) {
-  var end = randomType == constants.Randomize.ANCHOR_LAST ?
-            arr.length - 1 : arr.length;
+  var end = arr.length;
+  if (randomType === constants.Randomize.ANCHOR_LAST)
+    end = arr.length - 1;
+  if (randomType === constants.Randomize.ANCHOR_LAST_TWO)
+    end = arr.length - 2;
```
Comment (adrifelt) [inline extension/surveys/survey-generator.js]: same comment here: you also need to update flipArray

**Comment by moduloprime** [comment]
@adrifelt PTAL

File: extension/surveys/survey-generator.js [hunk 2]
```diff
@@ -473,16 +475,25 @@ function coinToss() {
...
+    end = arr.length - 1;
+  if (anchor === constants.Randomize.ANCHOR_LAST_TWO)
+    end = arr.length - 2; 
   var reverseArr = [];
   for (var i = end - 1; i >= 0; i--)
     reverseArr.push(arr[i]);
-  if (anchorLast)
+  if (anchor === constants.Randomize.ANCHOR_LAST)
     reverseArr.push(arr[arr.length - 1]);
+  if (anchor === constants.Randomize.ANCHOR_LAST_TWO) {
```
Comment (adrifelt) [inline extension/surveys/survey-generator.js]: else if?

File: extension/surveys/survey-generator.js [hunk 3]
```diff
@@ -473,16 +475,25 @@ function coinToss() {
...
+ *     two, or no elements.
  * @returns {Array.Object} The flipped array.
  */
-function flipArray(arr, anchorLast) {
-  var end = anchorLast ? arr.length - 1 : arr.length;
+function flipArray(arr, anchor) {
+  var end = arr.length;
+  if (anchor === constants.Randomize.ANCHOR_LAST)
+    end = arr.length - 1;
+  if (anchor === constants.Randomize.ANCHOR_LAST_TWO)
```
Comment (adrifelt) [inline extension/surveys/survey-generator.js]: else if?

**Comment by adrifelt** [comment]
P.S. you should respond to comments with "Done" once you've done them

File: extension/surveys/survey-generator.js (hunk 1 above)
Comment (moduloprime) [inline extension/surveys/survey-generator.js]: Done.

**Comment by adrifelt** [comment]
I'm in a bit of a hurry so I'm going to land this PR and then make the remaining changes in a separate PR

**Comment by moduloprime** [comment]
Cool, thx

**Pull request merged**